package.json

```diff
@@ -31,18 +31,18 @@
     "@react-navigation/material-top-tabs": "^5.2.19",
     "@react-navigation/native": "^5.7.6",
     "@react-navigation/stack": "^5.9.3",
-    "@sentry/react-native": "^3.3.3",
+    "@sentry/react-native": "^3.1.1",
     "@zulip/shared": "^0.0.14",
     "base-64": "^1.0.0",
     "blueimp-md5": "^2.10.0",
     "color": "^4.0.1",
     "date-fns": "^2.23.0",
-    "expo": "^44.0.0",
-    "expo-apple-authentication": "^4.0.0",
-    "expo-application": "~4.0.1",
-    "expo-screen-orientation": "~4.1.1",
-    "expo-sqlite": "^10.0.0",
-    "expo-web-browser": "10.1.1",
+    "expo": "^45.0.0",
```

Comment:
When we do the RN v0.68, we should study some special code that Expo
    wants us to add as part of that upgrade, beyond what the RN template
    app says. But that's a job for later, and I didn't see anything to
    do right now in this item.

What's the special code that Expo wants us to add as part of that upgrade? I don't think I totally follow.

If it is something we'll want to do as part of the RN upgrade, then we should mention it on the upgrade issue in our tracker.

(also nit in commit message: `AppDelegate.mm`, not `App.Delegate.mm`)

Reply:
Huh, this is strange. The quote in my commit message—

```
> Update your `App.Delegate.mm` (you should have moved from .m to
> .mm in the previous step) according to this diff
```

—doesn't actually match what's in the [Expo blog post](https://blog.expo.dev/expo-sdk-45-f4e332954a68#5954). From there:

> Update your `AppDelegate.mm` (you should have moved from `.m` to `.mm` in the previous step) update your Expo module configuration as done in [this diff](https://github.com/expo/expo/commit/b5764e22ad204fc44bed55cfeb38c78bb2bbdbcf).

I haven't yet managed to track down the thing I copy-pasted, and in fact the blog's current text [matches what it said in June](https://github.com/expo/expo/issues/17749#issuecomment-1166145217). 🤷

Since my quote failed to include the link on "this diff", I can't confirm whether I was looking at the same diff as the one linked in the actual blog post, so I'd better take another look at that.

Reply:
Probably the right thing to do is go through these:

```
$ git log --oneline --reverse upstream/sdk-44..upstream/sdk-45 -- templates/expo-template-bare-minimum
709dc40c7 Added android main file resolution  (#14964)
38181698b [templates] Update for latest SDK 44 and remove reanimated from bare template
b94d8a00e [templates] Remove reanimated babel plugin from bare template
b955ef624 [template] Remove RNGH import from index.js
69b878efb [autolinking] Introduce patcher for double-quoted import issue (#15655)
e842fb6c1 [autolinking] Fix errors from `expo_patch_react_imports!` (#15699)
8619d89df [fix] Invoke super methods in the AppDelegate.m to support ExpoAppDelegateSubscriber (#15684)
6126967d6 Reorder AppDelegate methods to prevent breaking behavior. (#15881)
6357d7dae Added platform gitignores (#15954)
60a411ee5 [template] Automatically apply android.packagingOptions from gradle.properties (#15863)
402679cac Upgrade to react-native 0.66 (#15914)
f5787adbc Upgrade to react-native 0.67 (#16038)
bae4b0eb1 [templates] update locks after bumping to RN 67 (#16397)
607fef4ab [android] remove jcenter (#16846)
abd66f4e8 Bump minimist in /templates/expo-template-bare-minimum (#16870)
b41c14cc1 [android][ios] Upgrade to react-native 0.68 (#16532)
7f2f41705 [all packages][Android] Migrate to `Android SDK 31` and `Java 11` (#16941)
ad83b875c [templates] remove lockfiles (#16975)
a3527a58b [android][ios] Upgrade react-native to 0.68.1 (#17063)
109ab9fcd Publish project templates
e95a20416 [templates] workaround expo-notifications broken when there's firebase sdk (#17201)
6b2c02fca [templates] add missing __apply_Xcode_12_5_M1_post_install_workaround (#17266)
fdddce11a [templates] Update bare SDK 45 template
6a62fea19 [build-properties] Introduce build-properties config-plugin (#17106)
4cb982247 [build-properties] Fix android build failure (#17321)
e0877701b [templates] Bump versions [skip ci]
e32e0d34e [templates] Update for release
adf446abf [android][ios] upgrade to react-native 0.68.2 (#17438)
cc0e84eb2 [templates] bump versions
6b95bbf60 [templates] Fix android.kotlinVersion support for expo-build-properties (#17625)
75e2cf7e3 [templates] Update outdated fresco versions on android (#17636)
79350c77f [templates] Support `pod install --project-directory=ios` (#17637)
875976b53 [build-properties][templates] add minSdkVersion support (#17647)
7f3099565 [templates] bump versions
```

Reply:
> Since my quote failed to include the link on "this diff", I can't confirm by commit ID that I was looking at the same diff as the one linked in the actual current blog post, so I'd better take another look at that.

It looks like it's the same link in the blog post today as it is in that quote from it from June. So I think we can pretty safely assume that when you read it in September, it was the same link then too.

It is *conceivable* that they changed it, then changed it back. But that seems awfully low probability compared to many other ways it's possible for us to get things wrong.

Reply:
> What's the special code that Expo wants us to add as part of that upgrade? I don't think I totally follow.
> 
> If it is something we'll want to do as part of the RN upgrade, then we should mention it on the upgrade issue in our tracker.

Done; see the two adjacent comments https://github.com/zulip/zulip-mobile/issues/5344#issuecomment-1322954035 and https://github.com/zulip/zulip-mobile/issues/5344#issuecomment-1324174190